README.md

```diff
@@ -27,24 +27,17 @@ Iroha is _Crash Fault Tolerant_ and has its own consensus algorithm - [YAC](http
 
 ## Documentation
 
-Our documentation is hosted at ReadTheDocs service here: [http://iroha.readthedocs.io](http://iroha.readthedocs.io).
-We have documentation in several languages available and you are welcome to contribute on [POEditor website](https://poeditor.com/join/project/SFpZw7o33o)!
+Our documentation is hosted at ReadTheDocs service here: [http://iroha.readthedocs.io](http://iroha.readthedocs.io) and has different versions.
```

Comment:
I think that "our documentation has different version" can have several meanings. Maybe rephrase it in a more specific way?

Reply:
Does it look better now, @lebdron ?

Reply:
Yes, thank you!